Formula/z.lua.rb

```diff
@@ -1,8 +1,8 @@
 class ZLua < Formula
   desc "New cd command that helps you navigate faster by learning your habits"
   homepage "https://github.com/skywind3000/z.lua"
-  url "https://github.com/skywind3000/z.lua/archive/1.8.11.tar.gz"
-  sha256 "213ea4c6287fa2952fb17db3191dcbaee014b28bff5230f615b23a1869aad912"
+  url "https://github.com/skywind3000/z.lua/archive/2.8.12.tar.gz"
+  sha256 "5092f186bf23e92437f3fed8bc5862a1e92ebfa322b7aed56887af9029bf6770"
```

Comment:
This looks like a typo in a major version in a tag.
Can we check with upstream if they really want to do it

Reply:
Actually, I've asked in https://github.com/skywind3000/z.lua/issues/130. Let's wait for a reply